cics-java-liberty-jdbc-web/src/main/webapp/scripts/employee.js

```diff
@@ -28,7 +31,8 @@ async function loadEmployees() {
         }
 
         const employees = await response.text();
-        employees.split("\n")
+        employees.split("
+")
```

Comment:
seems like the \n has been recognised as an actual new line which is interesting. change this back to how it was please before merging